File: src/UsersService/UsersService.Domain/Models/Company.cs
namespace UsersService.Domain.Models
{
    public class Company
    {
        public Guid Id { get; set; }
        public DateTime CreatedAt { get; set; }
        public Guid UserId { get; set; }
        public string LogoPath { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string City { get; set; }
        public string Address { get; set; }
        public string Email { get; set; }
        public string Phone { get; set; }
        public string WebSite { get; set; }
        public string Type { get; set; }
    }
}

File: frontend/src/app/core/models/company.ts
export default interface Company {
  id: string;
  userId: string;
  name: string;
  logoPath: string;
  description: string;
  city: string;
  address: string;
  email: string;
  phone: string;
  webSite: string;
  type: string;
  createdAt: Date;
}
